File: atcoder.jp/contests/arc012/tasks/arc012_1/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];

const solve = (day: string) => {
  if (day === "Monday") return 5;
  if (day === "Tuesday") return 4;
  if (day === "Wednesday") return 3;
  if (day === "Thursday") return 2;
  if (day === "Friday") return 1;
  return 0;
};

console.log(solve(input()));

File: atcoder.jp/contests/arc012/tasks/arc012_1/main.rs
use proconio::input;

fn main() {
    input! {
        day: String,
    }

    let ans = match day.as_str() {
        "Monday" => 5,
        "Tuesday" => 4,
        "Wednesday" => 3,
        "Thursday" => 2,
        "Friday" => 1,
        _ => 0,
    };

    println!("{}", ans);
}
